gcloud-java-pubsub/baseline/src/main/java/com/google/gcloud/pubsub/spi/v1/SubscriberApi.java

```diff
@@ -66,9 +65,26 @@
  */
 @javax.annotation.Generated("by GAPIC")
 public class SubscriberApi implements AutoCloseable {
+  // ========
```

Comment:
Per earlier discussion (I don't remember the status). Are these comments really necessary for the code-generation part? (This is not a common practice).